src/streaming/controllers/BufferController.js

```diff
@@ -306,6 +300,34 @@ function BufferController(config) {
         }
     }
 
+    function adjustSeekTarget () {
+        // Check buffered data only for audio and video
+        if (type !== Constants.AUDIO && type !== Constants.VIDEO) return;
+        if (isNaN(seekTarget)) return;
+
+        const segmentDuration = representationController.getCurrentRepresentation().segmentDuration;
```

Comment:
Suggest to move this after 313 as we might return without needing this

Reply:
done